src/cohere/utils.py

```diff
@@ -31,8 +32,8 @@ def get_validation_status(awaitable: typing.Union[EmbedJob, DatasetsGetResponse]
 
 def get_job(cohere: typing.Any,
             awaitable: typing.Union[CreateEmbedJobResponse, DatasetsCreateResponse, EmbedJob, DatasetsGetResponse]) -> \
-typing.Union[
-    EmbedJob, DatasetsGetResponse]:
+        typing.Union[
```

Comment:
is this indentation correct?

Reply:
it is ok but not pretty, will follow up with better formatting config across the handwritten code